Formula/influxdb.rb

```diff
@@ -23,6 +23,11 @@ def install
     revision = `git rev-parse HEAD`.strip
     version = `git describe --tags`.strip
 
+    # "main.version" must have version number only.
+    if version[0] == "v"
+      version = version[1..-1]
+    end
```

Comment:
They would be affected though since currently `version` is set to
```
version = `git describe --tags`.strip
```
for stable and HEAD alike in this formula.